client/actions/index.js

```diff
@@ -1,7 +1,12 @@
+import {request} from 'superagent'
+
 export const SHOW_ERROR = 'SHOW_ERROR'
 export const CLEAR_ERROR = 'CLEAR_ERROR'
 export const SHOW_SUCCESS = 'SHOW_SUCCESS'
 
+export const REQUEST_COMPANYINFO = 'REQUEST_COMPANYINFO'
+export const RECEIVE_COMPANYINFO = 'RECEIVE_COMPANYINFO'
```

Comment:
These action types are already defined in `actions/companies.js`. We should remove these and ensure that any file that needs these constants are getting them from `companies.js`.